lib/topological_inventory/ansible_tower/operations/worker.rb

```diff
@@ -0,0 +1,57 @@
+require "topological_inventory/ansible_tower/logging"
+
+module TopologicalInventory
+  module AnsibleTower
+    module Operations
+      class Worker
+        include Logging
+
+        def initialize(messaging_client_opts = {})
+          self.messaging_client_opts = default_messaging_opts.merge(messaging_client_opts)
+        end
+
+        def run
+          # Open a connection to the messaging service
+          require "manageiq-messaging"
+          client = ManageIQ::Messaging::Client.open(messaging_client_opts)
+
+          logger.info("Topological Inventory AnsibleTower Operations worker started...")
+          client.subscribe_messages(queue_opts) do |messages|
+            messages.each { |message| process_message(message) }
+          end
+        ensure
+          client&.close
+        end
+
+        private
+
+        attr_accessor :messaging_client_opts
+
+        def process_message(message)
```

Comment:
it can call `Processor.process` in this PR, it's generic code too

Reply:
:+1: added with specs